docs/installing/adding_devices.md

```diff
@@ -269,11 +280,12 @@ Now that your edge device is prepared, continue on to [Installing the agent](reg
 ### Hardware requirements
 {: #hard-req-mac}
 
-* 2010 or later 64-bit {{site.data.keyword.intel}} Mac device
-* MMU virtualization is required.
-* MacOS X version 10.11 ("El Capitan") or later
-* An internet connection for your machine (wired or wifi)
-* (optional) Sensor hardware: Many {{site.data.keyword.horizon}} edge services require specialized sensor hardware.
+- 2010 or later 64-bit {{site.data.keyword.intel}} Mac device
```

Comment:
Do we need to specify Intel-only Macs right now, and not arm-based Macs?

Reply:
I took a separate todo yesterday to open a ticket about doc changes to clarify that Open Horizon supports Intel-only Macs, not arm-based Macs. This is a RHEL ticket so I won't fix that here. Defer to https://github.com/open-horizon/open-horizon.github.io/issues/262